xarray/tests/test_backends.py

```diff
@@ -2243,60 +2243,53 @@ def open(self, path, **kwargs):
+        expected = xr.Variable(('TSTEP', 'LAY', 'ROW', 'COL'), data,
+                               dict(units='ppm', long_name='O3'.ljust(16),
+                                    var_desc='O3'.ljust(80)))
+        actual = camxfile.variables['O3']
+        assert_allclose(expected, actual)
+
+        data = np.array(
+            ['2002-06-03T00:00:00.000000000'],
+            dtype='datetime64[ns]'
+        )
+        expected = xr.Variable(('TSTEP',), data,
+                               dict(bounds='time_bounds',
+                                    long_name=('synthesized time coordinate ' +
+                                               'from SDATE, STIME, STEP ' +
+                                               'global attributes')))
+        actual = camxfile.variables['time']
+        assert_allclose(expected, actual)
+
     def save(self, dataset, path, **kwargs):
         dataset.to_netcdf(path, engine='netcdf4', **kwargs)
 
-    def test_weakrefs(self):
-        example = Dataset({'foo': ('x', np.arange(5.0))})
-        expected = example.rename({'foo': 'bar', 'x': 'y'})
-
-        with create_tmp_file() as tmp_file:
-            example.to_netcdf(tmp_file, engine='netcdf4')
-            on_disk = open_dataset(tmp_file, engine='pseudonetcdf')
-            actual = on_disk.rename({'foo': 'bar', 'x': 'y'})
-            del on_disk  # trigger garbage collection
-            assert_identical(actual, expected)
-
-    @pytest.mark.skip(reason='cannot pickle file objects')
-    def test_pickle(self):
-        pass
-
-    @pytest.mark.skip(reason='cannot pickle file objects')
-    def test_pickle_dataarray(self):
-        pass
-
-    @pytest.mark.skip(reason='does not support boolean dtype')
+    @pytest.mark.xfail(reason='https://github.com/pydata/xarray/issues/2061')
     def test_roundtrip_boolean_dtype(self):
-        # pnc does not support boolean dtype
         pass
 
-    @pytest.mark.skip(reason='does not support auto masking and scaling')
+    @pytest.mark.xfail(reason='https://github.com/pydata/xarray/issues/2061')
     def test_roundtrip_mask_and_scale(self):
-        # pnc does not support auto masking and scaling
         pass
 
-    @pytest.mark.skip(reason='does not support object types')
+    @pytest.mark.xfail(reason='https://github.com/pydata/xarray/issues/2061')
     def test_roundtrip_object_dtype(self):
-        # pnc does not support object types
         pass
 
-    @pytest.mark.skip(reason='encoding types are not consistent with xarray')
+    @pytest.mark.xfail(reason='https://github.com/pydata/xarray/issues/2061')
     def test_roundtrip_string_encoded_characters(self):
-        # pnc string treatment does not meet xarray expectations
-        # working on the right approach
         pass
 
-    def test_roundtrip_test_data(self):
-        expected = create_test_data()
-        with self.roundtrip(expected) as actual:
-            # dim3 sting type is <U instead of <U1
```

Comment:
The issue with `U` vs `U1` dtypes is something else, actually unrelated to this change. I'm going to try to fix it over in https://github.com/pydata/xarray/pull/2058

Reply:
So, skip for now?